### ISSUE TITLE
Wrong version number under Linux for 2.10 shows as Version: 2.9.1 Build id: 20230609-1009 (IEP-1001)

### ISSUE THREAD

**Issue by Maksons** [espressif/idf-eclipse-plugin#797]
**Describe the bug**
Version number of Linux package from 2.10 release show wrong number under Help -> About Espressif-IDE : 
```
Espressif-IDE is an Integrated Development Environment(IDE) based on Eclipse CDT for developing IoT Applications using ESP-IDF framework.
Version: 2.9.1
Build id: 20230609-1009
Copyright (c) 2021 ESPRESSIF SYSTEMS (SHANGHAI) CO., LTD. All rights reserved.
A clear and concise description of what the bug is.
```

**To Reproduce**
1. Get https://dl.espressif.com/dl/idf-eclipse-plugin/ide/Espressif-IDE-2.10.0-linux.gtk.x86_64.tar.gz
2. Run it
3. Got to menu "Help -> About Espressif-IDE"

**Comment by kolipakakondal** [comment]
Thanks @Maksons let us verify.

@AndriiFilippov Could you please help me in verifying this?

**Comment by kolipakakondal** [comment]
We have released v2.11.0 on 21st sept 2023, it should reflect the correct version, hence closing this issue
https://github.com/espressif/idf-eclipse-plugin/releases/tag/v2.11.0
